tests/search/share-modal.test.js

```diff
@@ -0,0 +1,37 @@
+import { ClientFunction } from 'testcafe';
+import Search from './modals/Search';
+
+const getWindowLocation = ClientFunction(() => window.location);
```

Comment:
let's move this file to tests/search (tests/search/share-modal.test.js) so it sits along side base-search page.test.js